modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ItemActions.js

```diff
@@ -101,6 +101,9 @@ export default function ItemActions({item}) {
 			<ClayDropDown
 				active={active}
 				alignmentPosition={Align.BottomRight}
+				menuElementAttrs={{
+					className: 'cadmin',
```

Comment:
I think this is way too implicit. The fact that simply adding a cssclass produces all this is at the very least unexpected. If we go this way, I think should be explicit about it and not rely on css classes to toggle the isolation on or off.

Reply:
I don't know how wrapping this in a declarative tag would easily work. ClayDropDown renders the dropdown-menu in a React Portal. Wrapping this in a `layout:isolate` tag would isolate the toggle button, but the dropdown-menu would be un-isolated at the bottom of the page. This applies to tooltips, popovers, modals, etc.

We have mixed content stuff like `portlet-topper`, `page-editor__topper`, and `page-editor__editable` that can't be easily scoped by wrapping it in a tag.

Tags would work for admin components that are completely separate from user content, but not for mixed. This means we need to learn two syntaxes depending on the situation.